cico_functions.sh

```diff
@@ -33,6 +33,30 @@ function load_jenkins_vars() {
   fi
 }
 
+function check_version() {
```

Comment:
@Prabhav-Thali could you please make the function name more descriptive. For me it is not clear what exactly 'check_version' does

Reply:
Updated the function name and added docs for the same.